rancher2/schema_cluster.go

```diff
@@ -712,6 +712,13 @@ func clusterFields() map[string]*schema.Schema {
 			Description: "Windows preferred cluster",
 			ForceNew:    true,
 		},
+		"fleet_workspace": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Default:     "fleet-default",
```

Comment:
Due to `fleet_workspace ` is a new argument on `rancher2_cluster`, i think'd be more secure for tfp upgrade, set this argument as `Computed: true` and removing default value.